src/JustEat.StatsD/IStatsDUdpClient.cs

```diff
@@ -1,11 +1,6 @@
-﻿using System;
-using System.Collections.Generic;
-
-namespace JustEat.StatsD
+﻿namespace JustEat.StatsD
 {
-    public interface IStatsDUdpClient : IDisposable
+    public interface IStatsDUdpClient : IStatsDClient
```

Comment:
This is still a breaking change because the inheritance hierarchy is being changed. In-place updates of the assembly wouldn't work, even if re-compiling the consuming code after updating still works.